workflow/rules/eval.smk

```diff
@@ -60,15 +60,23 @@ rule remove_non_pass:
         "v3.3.6/bio/bcftools/view"
 
 
-use rule normalize_truth as normalize_calls with:
+rule normalize_calls:
     input:
-        "results/filtered-variants/{callset}.bcf",
+        bcf="results/filtered-variants/{callset}.bcf",
         ref="resources/reference/genome.fasta",
         ref_index="resources/reference/genome.fasta.fai",
+        regions=get_target_regions,
     output:
         "results/normalized-variants/{callset}.vcf.gz",
     log:
         "logs/normalize-calls/{callset}.log",
+    conda:
+        "../envs/tools.yaml"
+    shell:
+        "(bedtools intersect -b {input.regions} -a "
+        "<(bcftools view {input.bcf}) -wa -f 1.0 -header | "
+        "bcftools norm --atomize --check-ref s --fasta-ref {input.ref} --rm-dup exact | "
```

Comment:
Maybe we need to change back to using params and get_norm_params for this part?

Reply:
Yes, get_norm_params contains an extra parameter for test cases where limit_reads is true